src/routes/setting/setting.js

```diff
@@ -80,9 +80,11 @@ const form = ({
         valuePropName = 'value'
         break
     }
+    // This part of the code will be deleted when the backend is removed and the setting is changed
+    let temporarilyHideSettings = setting.id === 'guaranteed-engine-cpu'
```

Comment:
You can directly check the type of this setting. The type here becomes `deprecated`:
```
{
"actions": { },
"definition": {
"category": "danger Zone",
"default": "",
"description": "This setting is replaced by 2 new settings \"Guaranteed Engine Manager CPU\" and \"Guaranteed Replica Manager CPU\" since Longhorn version v1.1.1. \nThis setting was used to control the CPU requests of all Longhorn Instance Manager pods. \n",
"displayName": "Guaranteed Engine CPU (Deprecated)",
"readOnly": true,
"required": false,
"type": "deprecated",
},
"id": "guaranteed-engine-cpu",
"links": {
"self": "…/v1/settings/guaranteed-engine-cpu",
},
"name": "guaranteed-engine-cpu",
"timestamp": "2021-03-08T06:18:23.426111167Z",
"type": "setting",
"value": "",
}
```

Reply:
ok